interface/views.py

```diff
@@ -149,19 +149,18 @@ def submission_list(request):
 @login_required
 def submission_result(request, pk):
     sub = get_object_or_404(Submission, pk=pk)
+    show_review_message = True if not sub.review_message.startswith('none') else False  # noqa: E501
```

Comment:
This is silly. The default for `review_message` should be an empty string and it should have `allow_empty=True`. `none` is not a great default.

https://github.com/vmck/acs-interface/blob/master/interface/models.py#L75